index.html

```diff
@@ -13,5 +13,16 @@
     <img class="title-image" src="assets/sectionTitleImage.svg" alt="image-title"></img>
     <h1 class="title-text">Front End Tools</h1>
   </div>
+  <br>
+  <div class="main-tech-container">
+    <div class="background-tech-container">
```

Comment:
Make this div self-closing: `<div class="background-tech-container"/>`

Reply:
Done